.github/workflows/versions.yml

```diff
@@ -21,36 +21,39 @@ on:
   workflow_call:
     inputs:
       version:
-        description: 'New version to use for the extension. Example: 1.5.2'
+        description: 'The version to update or validate for the extension (ex: 1.2.3).'
         required: true
         type: string
 
       branch:
-        description: 'The target branch where the version update will be applied and the pull request will be merged into.'
+        description: |
+          The target branch used for both version validation and updates. 
+          For the update action, this is also the branch into which the pull request will be merged.
         required: true
         type: string
 
       dependencies:
         description: |
           A comma-separated list of dependencies with their versions. 
-          Each dependency can optionally specify the file paths where it applies using the `@` symbol.
+          Each dependency can optionally specify a semicolon-separated list of the file paths where it applies using the `@` symbol.
           
-          **Syntax:** `<name> <version>[@file_path1[,file_path2,...]]`
+          **Syntax:** `<name> <version>[@file_path1[;file_path2;...]]`
 
           - If the `@` syntax is used, the dependency will only apply to the specified files.
-          - When specifying custom files, you must provide either an absolute or relative path to each file.
           - If the `@` symbol is omitted, the dependency applies to all relevant files.
+          - When specifying custom file paths, you must provide either an absolute or relative path to each file.
+          - If a dependency is missing a version, it will be skipped.
 
           **Example:**
           iOS: `"AEPCore 3.1.1, AEPServices 8.9.10@AEPCore.podspec, Edge 3.2.1@Package.swift"`
-          Android: `"AEPCore 7.8.9, AEPEdgeIdentity 8.9.10@code/gradle.properties"`
+          Android: `"AEPCore 7.8.9, AEPEdgeIdentity 8.9.10@code/gradle.properties;code/example.kt"`
```

Comment:
why are we adding `code/example.kt` is it also an intended sample? If yes we can call it `code/constants.kt` for better context. If we want to add this let's add the sample for iOS as well.

Reply:
we can also give full samples like on L63